src/modules/flow/py-formatting/Kconfig

```diff
@@ -0,0 +1,17 @@
+config FLOW_NODE_TYPE_PY_FORMATTING
+	tristate "Node type: py-formatting"
```

Comment:
the name is too large and the "py-" is not useful in the name. You can call it "format" or "formatting" (I prefer the short version), in its description you can mention the syntax is based on the new Python formatting for those that are used to it. But this is our formatting and we have control about that, if we need further extensions or restrictions.

This also mean we should document this on our own as well, it can be based on Python's docs and even point to them, but we need something at our side.